cli/request.go

```diff
@@ -43,5 +45,9 @@ func extendRequestWithContent(fs filesystemspec.FileSystem, req controller.Reque
 		req.Units = append(req.Units, controller.Unit{Name: name, Content: content})
 	}
 
+	if len(req.Units) == 0 {
+		return controller.Request{}, errgo.Newf("No unit files found for group '%s'", req.Group)
+	}
+
```

Comment:
the code later checks on `req.Units[0]` and throws a panic, even before the controller is reached. So I would say yes.